indico/web/client/js/react/forms/fields.jsx

```diff
@@ -70,6 +71,10 @@ export function FormFieldAdapter({
     delete input.checked;
   }
 
+  if (loaderWhileValidating) {
```

Comment:
Probably not worth re-opening just for this. But isn't this (in the props) equivalent?
```
loading={loaderWhileValidating && validating}
```

Reply:
No, some fields (the react-dates date picker) complain about any unexpected prop in their proptypes validation. I had `loading={loaderWhileValidating ? validating : undefined}` initially hoping that `undefined` props would simply not be passed along, but this wasn't the case...